tools/generate-package-lists

```diff
@@ -8,9 +8,13 @@ import os
 import subprocess
 import sys
 import time
+import requests
 
 PROJECT_ROOT=os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 DISTRIBUTION_DIR=os.path.join(PROJECT_ROOT, 'distributions')
+r = requests.get("https://raw.githubusercontent.com/osbuild/osbuild-composer/main/dnf-json")
+with open(os.path.join(PROJECT_ROOT, "dnf-json"), 'w') as f:
```

Comment:
When moved below, could just use the DNF_JSON constant instead of the path.join.